### PR TITLE
Added new user flag when user registers with facebook

### PR DESCRIPTION
**Pull request by TheHollidayInn**
Fixes put_issue_url_here
### Changes

When a new user registers with Facebook, the api will not return a newUser value of true. This is used in the app to tell when a user is registering or logging in via social media. 

---

UUID:

### CODE REVIEW

File: website/server/controllers/api-v3/auth.js [hunk 1]
```diff
@@ -162,8 +162,10 @@ api.registerLocal = {
 };
 
 function _loginRes (user, req, res) {
+  var newUser = false;
```
Comment (crookedneighbor) [inline website/server/controllers/api-v3/auth.js]: You can just do `var isNewUser = user.newUser === true`

**Comment by crookedneighbor** [comment]
There should be test coverage around this. Other than that, seems sound to me.

File: website/server/controllers/api-v3/auth.js (hunk 1 above)
Comment (paglias) [inline website/server/controllers/api-v3/auth.js]: Should be let not var

Comment (paglias) [inline website/server/controllers/api-v3/auth.js]: Actually you don't need to create a new car at all. You can just pass user.newUser || false

Comment (crookedneighbor) [inline website/server/controllers/api-v3/auth.js]: +1 to that

**Comment by TheHollidayInn** [comment]
Updated!

**Comment by paglias** [comment]
@TheHollidayInn looks good to me 👍

**Pull request merged**